CMakeLists.txt

```diff
@@ -99,6 +99,9 @@ if(PNG_FOUND)
   set(ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES} ${PNG_LIBRARIES})
 endif(PNG_FOUND)
 
+find_library(M_LIB m)
```

Comment:
that's only relevant on unix, maybe add only set ADDITIONAL_LIBRARIES if M_LIB is valid